src/nunit-gui/Views/SettingsPage.cs

```diff
@@ -86,13 +80,19 @@ public virtual void ApplySettings()
 
         #endregion
 
+        void LoadAndSetSettings()
```

Comment:
An error on my part. I wanted to make sure that the flag was always set when overriding `LoadSettings`. `SettingsLoaded` is now set in `OnLoad`.